grommet-leaflet/package.json

```diff
@@ -10,7 +10,8 @@
   "license": "ISC",
   "main": "src/index.js",
   "scripts": {
-    "lint-fix": "eslint src --fix"
+    "lint": "eslint src/**/*.js src/**/*.jsx",
```

Comment:
In grommet, we just have `eslint src` -- is there something we can add to our eslintrc to have it lint jsx pages too?

Reply:
I couldn;t find anything that we could add to the eslintrc file, but I did find a better way to write the command in the package.json files